aptos-move/dynamic-transaction-composer/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "aptos-dynamic-transaction-composer"
+description = "Generating Move Script from a batched Move calls"
+version = "0.1.0"
+
+# Workspace inherited keys
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+publish = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+anyhow = { workspace = true }
+bcs = { workspace = true }
+getrandom = { workspace = true, features = ["js"] }
+hex = { workspace = true }
+move-binary-format = { workspace = true }
+move-bytecode-verifier = { workspace = true }
+move-core-types = { workspace = true }
+reqwest = { version = "*", features = ["blocking"] }
```

Comment:
The reason why I'm avoiding workspace is this repo will be compiled into WASM and thus might have different dependencies and I don't want to pollute the main workspace because of that.

Reply:
Yeah, I started to think how we deal with THAT. As a matter of fact, you have to pull in significant other parts of the code which already have workspace dependencies. We need a more general solution for this problem. I'm not sure whether there is something like 'dependency override' in cargo. Or perhaps for the WASM build, we need to generate the build config by our own script, that is generate source for a crate then compile it. For now I think this should be workspace too, it doesn't appear to solve the problem this way.

Reply:
Still needs resolution. Not using workspace doesn't help here. Also "*" is not our practice, should be a concrete version.